addons/common/XEH_preClientInit.sqf

```diff
@@ -8,4 +8,7 @@ if (hasInterface) then {
     GVAR(actionListUpdated) = false; // Set true to force recreation of actions.
     GVAR(nextActionIndex) = 0; // Next index that will be given out.
     GVAR(actionListPFEH) = false;
+
+    // Add CBA_MiscItems to VirtualArsenal
+    [missionnamespace, "arsenalOpened", {call CBA_fnc_handleArsenalOpened}] call bis_fnc_addscriptedeventhandler;
```

Comment:
Check camelCase capitalization in:
XEH_preClientInit - line 13.
fnc_handleArsenalOpened - lines 13, 19, 21, 30-44, 51-57 and 60.

Reply:
```sqf
[missionNamespace, "arsenalOpened", {call CBA_fnc_handleArsenalOpened}] call BIS_fnc_addScriptedEventHandler;
```